hwy/ops/generic_ops-inl.h

```diff
@@ -1615,6 +1615,19 @@ HWY_API V AESLastRoundInv(V state, const V round_key) {
   return state;
 }
 
+template <uint8_t kRoundConstant, class V>  // u8
```

Comment:
Consider adding `HWY_IF_U8_D(DFromV<V>)` to ensure this doesn't compile (with clear explanation) if the type is not u8?